custom_components/midea_ac_lan/config_flow.py

```diff
@@ -270,11 +267,13 @@ async def async_step_discovery(
             for device_id, device in self.devices.items():
                 # remove exist devices and only return new devices
                 if not self._already_configured(
-                    str(device_id), device[CONF_IP_ADDRESS]
+                    str(device_id),
+                    device[CONF_IP_ADDRESS],
                 ):
-                    self.available_device[device_id] = (
-                        f"{device_id} ({self.supports.get(device.get(CONF_TYPE))})"
-                    )
+                    self.available_device[
+                        device_id
+                    ] = f"{device_id} ({self.supports.get(
+                            device.get(CONF_TYPE))})"
```

Comment:
Python version 3.11 does not allow new lines in expression parts of non-triple-quoted f-strings

Reply:
Do this in later PR